Public/Src/Cache/ContentStore/Distributed/NuCache/LocalLocationStoreConfiguration.cs

```diff
@@ -203,32 +205,87 @@ public DistributedCentralStoreConfiguration(AbsolutePath cacheRoot)
         public int MaxSimultaneousCopies { get; set; } = 10;
     }
 
+    /// <summary>
+    /// Provides Azure Storage authentication options for <see cref="BlobCentralStorage"/>
+    /// </summary>
+    public class AzureBlobStorageCredentials
+    {
+        /// <nodoc />
+        public readonly string ConnectionString;

```

Comment:
I meant to use get-only property, not readonly fields.

Readonly fields are fine, but we usually prefer using get-only properties in case we'll need to change the implementation without changing the clients of the code.